pkg/services/preference/pref.go

```diff
@@ -8,6 +8,6 @@ type Service interface {
 	GetWithDefaults(context.Context, *GetPreferenceWithDefaultsQuery) (*Preference, error)
 	Get(context.Context, *GetPreferenceQuery) (*Preference, error)
 	Save(context.Context, *SavePreferenceCommand) error
-	Patch(ctx context.Context, cmd *PatchPreferenceCommand) error
+	Patch(context.Context, *PatchPreferenceCommand) error
```

Comment:
😁 ty these drive me crazy

Reply:
haha, I copy paste the signature, and that's how I introduce those :P